docs/layers.md

```diff
@@ -516,40 +929,55 @@ Also includes water `line` geometries for river and stream centerlines and "labe
 
 Mapzen calculates the composite exterior edge for overlapping water polygons and marks the resulting line `boundary=yes`. Set to `yes` when present on `line` geometry, or from Natural Earth line source.
 
-**Water properties (common):**
+#### Water properties (common):
 
 * `name`: including localized name variants
 * `kind`: detailed below, per geometry type
 * `source`: one of `naturalearthdata.com`, `openstreetmapdata.com`, `openstreetmap.org`
 * `boundary`: `yes`, on lines only. See description above.
 * `sort_key`: a suggestion for which order to draw features. The value is an integer where smaller numbers suggest that features should be "behind" features with larger numbers.
 
-**Water properties (common optional):**
+#### Water properties (common optional):
 
 * `area`: in square meters (spherical Mercator, no real-world), `polygon` features only
 * `id`: OpenStreetMap feature `osm_id`, when sourced from `openstreetmap.org`
 * `is_tunnel`: for `line` features only (`yes` values only)
 
-**Water kind values (point, polygon):**
+#### Water kind values (point, polygon):
 
-* `basin`, `dock`, `lake`, `ocean`, `playa`, `riverbank`, `swimming_pool`, and `water`
+* `basin`
+* `dock`
+* `lake`
+* `ocean`
+* `playa`
+* `riverbank`
+* `swimming_pool`
+* `water`
 
 Additionally, a `reservoir=yes` or `alkaline=yes` value can be present on the appropriate `kind=lake` features.
 
-**Water kind values (point only):**
+#### Water kind values (point only):
 
 These are intended for label placement, and are included as points only.
 
-* `bay`, `fjord`, `strait`, `sea`
+* `bay`
+* `fjord`
+* `strait`
+* `sea`
 
-    Gotchas:
+**Gotchas:**
 
-    * `Alkaline Lake` and `Playa` only exist in Natural Earth. Zooming in, your feature may disappear. Beware the desert around Great Salt Lake in Utah!
-    * `ocean` and `Ocean` are equivalent, **both** should be used in filters.
-    * `reservoir` and `Reservoir` are equivalent, **both** should be used in filters.
-    * Some of the minor kinds (like `bay`, `strait`, and `fjord`) are used for label_placement points only, as their area would duplicate water polygons already present from openstreetmapdata.com.
-    * The (capitalized) Natural Earth values don't seem to be coming thru, but should be, instead they're being grouped into `lake`, `playa`, and `ocean`.
+* `Alkaline Lake` and `Playa` only exist in Natural Earth. Zooming in, your feature may disappear. Beware the desert around Great Salt Lake in Utah!
```

Comment:
Yes we did, good catch! #689 did update the kind list appropriately, but did not update the Gotchas section (the section commented on above). My bad since, I was the reviewer. I'll fix here.